README.md

```diff
@@ -247,6 +247,10 @@ These variables are the defaults of our roles, if you want to override the prope
 | smtp_port                      | Port number of SMTP server                                                                                       |                                             |
 | smtp_user                      | Username for SMTP authentication                                                                                 |                                             |
 | smtp_password                  | Password for SMTP authentication                                                                                 |                                             |
+| enable_oidc                    | Enable OIDC integration, this will disable the username/password authentication method                           | false                                       |
```

Comment:
Better to remove the `will`. `this disables the username/password ...

Reply:
done